README.md

```diff
@@ -17,7 +17,7 @@ urlFragment: contoso-chat-csharp-prompty
 
 # Contoso Chat Retail with .NET and Semantic Kernel
 
-Samples in JavaScript, Python, and Java. Learn more at https://aka.ms/azai.
+This sample demonstrates how to build, evaluate, and deploy, a retail copilot application end-to-end using Azure AI and Cosmos DB. It's built using .NET with the Semantic Kernel library and Prompty prompt files.
```

Comment:
Won`t we need to refer to other azai samples as well?

Reply:
Yeah, I think we should consider a pass over these readmes later and see about cross-links. I removed this link for now because it takes you to Python documentation, which isn't good for .NET devs, and I also wanted to have a quick intro to what the sample is right at the top of the readme.